### PR TITLE
Even More Kiosk UI Improvements

### LINKED ISSUE #1988: Automatically un-sticky the team on the kiosk
Wait for some amount of time (an hour?) with no taps.

### PR DESCRIPTION
**Pull request by RossIV**
- Adds color, dropshadow, and rounded corners to team buttons
image

- Stylistic improvements to the pending-swipe modal
image

- Adds a (very imperfect) sound test option for administrators
image

- Adds timer for pending-swipe modal for both non-sticky (30 seconds) and sticky (10 minutes) mode (close #1988)
- Adds visual progress bar for timed modals
- Buttons on SweetAlert modals are bigger for easier interaction on touchscreen

### CODE REVIEW

File: resources/js/components/attendance/AttendanceKiosk.vue [hunk 1]
```diff
@@ -172,29 +172,42 @@
...
                         cardImg.src = '/img/Universal_Contactless_Card_Symbol.svg';
-                        var cardDiv = document.createElement('div');
+                        let cardDiv = document.createElement('div');
                         cardDiv.className = 'animated-contactless-card';
                         cardDiv.appendChild(cardImg);
                         Swal.getTitle().parentNode.insertBefore(cardDiv, Swal.getTitle());
                     },
                     didDestroy: () => {
-                      let checkbox = document.getElementById('swal2-checkbox')
+                      let checkbox = document.getElementById('stick-checkbox')
```
Comment (evan10s) [inline resources/js/components/attendance/AttendanceKiosk.vue:210]: is `stick` a typo?

File: resources/js/components/attendance/AttendanceKiosk.vue [hunk 2]
```diff
@@ -172,29 +172,42 @@
...
                         document.activeElement.blur();
-                        document.getElementById('swal2-checkbox')
-                          .addEventListener("change", checkboxEventListener.bind(this));
+                        let checkbox = document.getElementById('stick-checkbox');
+                        checkbox.checked = this.stickToTeam;
+                        checkbox.addEventListener("change", checkboxEventListener.bind(this));
+
                         // Add animated contactless card symbol
-                        var cardImg = document.createElement('img');
+                        let cardImg = document.createElement('img');
```
Comment (evan10s) [inline resources/js/components/attendance/AttendanceKiosk.vue:202]: you can probably make this a `const`

File: resources/js/components/attendance/AttendanceKiosk.vue [hunk 3]
```diff
@@ -172,29 +172,42 @@
...
                         cardImg.src = '/img/Universal_Contactless_Card_Symbol.svg';
-                        var cardDiv = document.createElement('div');
+                        let cardDiv = document.createElement('div');
```
Comment (evan10s) [inline resources/js/components/attendance/AttendanceKiosk.vue:204]: same here

File: resources/js/components/attendance/AttendanceKiosk.vue [hunk 4]
```diff
@@ -295,13 +310,15 @@
                             } else if (response.data.users.length == 1 && response.data.users[0].roles.filter(role => role.name.toString() === "admin").length === 1) {
                                 // Roles retrieved and the user is an admin
                                 console.log('User is an admin!');
```
Comment (evan10s) [inline resources/js/components/attendance/AttendanceKiosk.vue:312]: can we remove this line?

**Review by evan10s — CHANGES_REQUESTED** [review]

**Comment by evan10s** [comment]
Functionality is all good in test

File: resources/js/components/attendance/AttendanceKiosk.vue (hunk 1 above)
Comment (RossIV) [reply resources/js/components/attendance/AttendanceKiosk.vue:210]: No, that's the correct `id` of the new checkbox. `stick` as in `stickToTeam`.

File: resources/js/components/attendance/AttendanceKiosk.vue (hunk 4 above)
Comment (RossIV) [reply resources/js/components/attendance/AttendanceKiosk.vue:312]: Is this referring to the admin log? Can't tell, GitHub is being silly.

**Review by evan10s — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**